src/rgw/rgw_s3select.cc

```diff
@@ -350,9 +350,9 @@ int RGWSelectObj_ObjStore_S3::run_s3select_on_csv(const char* query, const char*
 {
   int status = 0;
   uint32_t length_before_processing, length_post_processing;
-  csv_object::csv_defintions csv;
```

Comment:
This requires a fix in src/s3select/include/s3select.h that I'm not seeing in this change

Reply:
https://github.com/ceph/s3select/ is a submodule, so the change would need to merge there first

Reply:
I'll drop this for now. Once I've done most of the passes for ceph, people can point out the submodules and we can see about working on them.